lua/telescope/actions/set.lua

```diff
@@ -148,11 +148,19 @@ end
 ---@param direction number: The direction of the scrolling
 --      Valid directions include: "1", "-1"
 action_set.scroll_previewer = function (prompt_bufnr, direction)
+  local picker = action_state.get_current_picker(prompt_bufnr)
+
+  -- Check if we actually have a previewer
+  if (type(picker['previewer']) ~= "table" or picker.previewer['scroll_fn'] == nil) then
```

Comment:
can you do `picker.previewer` and `picker.previewer.scroll_fn`

Reply:
Done. Still new to lua ;)